docs/source/contributing/documentation.rst

```diff
@@ -76,7 +74,8 @@ In the example above, ``Formula1`` followed by ``.. manim::`` and is the scene t
 
 .. note::
 
-   Sometimes, when you reload an example in your browser, it has still the old website somewhere in its cache. If this is the case, delete the website cache, or open a new `incognito tab`` in your browser, then the latest docs should be shown. If this still doesn't work, you may need to delete the contents of ``docs/source/references``.
+   **Only for locally built documentation:** Sometimes, when you reload an example in your browser, it has still the old website somewhere in its cache. If this is the case, delete the website cache, or open a new `incognito tab`` 
+   in your browser, then the latest docs should be shown. If this still doesn't work, you may need to delete the contents of ``docs/source/references``.
```

Comment:
The first part (reloading documentation) applies to both documentation rendered by RTD and local documentation.

Only the second part ("If this still doesn't work, ...") applies to locally built documentation.

Reply:
Done